spec/support/corpus.rb

```diff
@@ -134,15 +138,33 @@ def install_mutant
         system(%w[bundle])
       end
 
-      # Not in the docs. Number from chatting with their support.
-      CIRCLE_CI_CONTAINER_PROCESSES = 2
+      # The effective ruby file paths
+      #
+      # @return [Array<Pathname>]
+      def effective_ruby_paths
+        paths = Pathname
+          .glob(repo_path.join(RUBY_FILE_PATTERN))
+          .sort_by(&:size)
+          .reverse
```

Comment:
I know this was in the previous code but I'm curious, why do you sort and reverse here?